R/spacy_parse.R

```diff
@@ -46,6 +47,7 @@ spacy_parse <- function(x,
                         lemma = TRUE,
                         entity = TRUE, 
                         dependency = FALSE,
+                        noun_phrase = FALSE,
```

Comment:
let's remove the underscore, since none of the other arguments have it. So just `nounphrase`.